tests/infer/mcmc/test_mcmc.py

```diff
@@ -51,6 +51,16 @@ def test_mcmc_interface():
     assert_equal(sample_std, torch.tensor([1.0]), prec=0.08)
 
 
+@pytest.mark.parametrize("num_chains", [1, 2])
+def test_mcmc_diagnostics(num_chains):
+    data = torch.tensor([1.0])
+    kernel = PriorKernel(normal_normal_model)
+    mcmc = MCMC(kernel=kernel, num_samples=10, num_chains=1).run(data)
```

Comment:
oppss, `num_chains=num_chains`, my brain is foggy recently I guess >"< srrrrrrrrrrrr